core/modules/widgets/reveal.js

```diff
@@ -90,10 +90,20 @@ RevealWidget.prototype.positionPopup = function(domNode) {
 			top = this.popup.top + this.popup.height;
 			break;
 	}
+
```

Comment:
Very minor point, but we try to avoid blank lines except for separating top level methods (there are two more below too).